cirq-core/cirq/sim/sparse_simulator_test.py

```diff
@@ -1270,6 +1301,20 @@ def test_nondeterministic_mixture_noise():
     assert result1 != result2
 
 
-def test_unsupported_noise_fails():
-    with pytest.raises(ValueError, match='noise'):
-        cirq.Simulator(noise=cirq.amplitude_damp(0.5))
+def test_noise_model():
+    q = cirq.GridQubit(0, 0)
+    circuit = cirq.Circuit(cirq.I(q), cirq.measure(q))
+
+    calibration = cirq_google.Calibration(_CALIBRATION_DATA)
+    noise_model = simple_noise_from_calibration_metrics(
+        calibration=calibration,
+        depol_noise=True,
+        readout_decay_noise=True,
+        readout_error_noise=True,
+    )
+    simulator = cirq.Simulator(noise=noise_model)
+
+    result1 = simulator.run(circuit, repetitions=50)
+    result2 = simulator.run(circuit, repetitions=50)
+
+    assert result1 != result2
```

Comment:
I suppose for this PR we just want to verify that the simulator accepts the noise model, as opposed to raising to an exception. However, we can make the test more useful by checking that the measurement falls into an interval (which ideally would be verified by a calculation independent of the code under test).